platform/security/handler/security-handler-pki/src/main/java/org/codice/ddf/security/handler/pki/CrlChecker.java

```diff
@@ -127,15 +124,15 @@ URL urlFromPath(String location) {
      * @param location location of properties file
      * @return Properties from the file
      */
-    Properties loadProperties(String location) {
+    static Properties loadProperties(String location) {
```

Comment:
❓ Same question.
(I noticed that those are used by the unit tests, but I don't think it's a good enough reason to expose them like we do, especially since they are so short. And depending on the implementation of the class under test to write the test feels like a smell).